HPXMLtoOpenStudio/resources/hvac.rb

```diff
@@ -4142,6 +4000,8 @@ def self.get_hpxml_hvac_systems(hpxml)
       heating_system = nil
       if is_central_air_conditioner_and_furnace(hpxml, cooling_system.attached_heating_system, cooling_system)
         heating_system = cooling_system.attached_heating_system
+      elsif is_ptac(hpxml, cooling_system.attached_heating_system, cooling_system)
+        heating_system = cooling_system.attached_heating_system
```

Comment:
I'm confused as to why you don't have to skip the PTACHeating system inside the heating system loop below (like we do for AC+furnace).

Reply:
I think it is because I didn't add a function like apply_ptac_heating() being called at `add_heating_system` so it just jumped out of the `add_heating_system` after system type checks and didn't create duplicated systems. But it's absolutely a bug because it calculates the sequential loads multiple times, so it will go wrong for multiple systems. Added to the `base-hvac-multiple.xml` and verified this bug.